File: VSC Files/Personal Things/Trash File/trashFile.c

int trashFile(){
    int sum = 0;
    printf("Starting...");
    for(int i=1;i<9223372036854775807;i++){
        float prevSum = sum;
        sum += 1.0/i;
        if(sum<prevSum) printf("We went over the limit whoops, that happened at i=" + i);
        if(i%1000000000==0) printf(sum + "\tat i = " + (i/1000000000) + " E10^9\t" + (9223372036854775807-i) + " away from our limit");
    }
    printf(sum);
    return 0;
}


// import java.util.*;

// public class trashFile{
//     public static void main(String[] args){


//         // System.out.println("Starting...");
//         // System.out.println("Max long value: " + Long.MAX_VALUE);
//         // double sum = 0;
//         // for(long i=1;i<Long.MAX_VALUE;i++){
//         //     double prevSum = sum;
//         //     sum += 1.0/i;
//         //     if(sum<prevSum) System.out.println("We went over the limit whoops, that happened at i=" + i);
//         //     if(i%1000000000==0) System.out.println(sum + "\tat i = " + (i/1000000000) + " E10^9\t" + (Long.MAX_VALUE-i) + " away from our limit");
//         // }

//         // System.out.println(sum);
//     }
// }

File: Personal Things/Trash File/trashFile.java

import java.util.*;

public class trashFile{
    public static void main(String[] args){

        ArrayList<Integer> list2 = new ArrayList<Integer>();
        for(int i=1;i<=10;i++){
            list2.add(i);
        }
        for(int i=0;i<list2.size();i++){
            if(list2.get(i)==3 || list2.get(i)==4)
                list2.remove(i);
            System.out.println(list2);
        }
        System.out.println(list2);
    }
}

















